sdk/test/ProtocolTests/Generated/QueryProtocol/QueryLists.cs

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/*
+ * Do not modify this file. This file is generated.
+ */
+using AWSSDK.ProtocolTests;
+using AWSSDK.ProtocolTests.Utils;
+using AWSSDK_DotNet35.UnitTests.TestTools;
+using Amazon.QueryProtocol;
+using Amazon.QueryProtocol.Model;
+using Amazon.QueryProtocol.Model.Internal.MarshallTransformations;
+using Amazon.Runtime;
+using Amazon.Runtime.Internal;
+using Amazon.Runtime.Internal.Transform;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+
+namespace AWSSDK.ProtocolTests.AwsQuery
+{
+    [TestClass]
+    public class QueryLists
+    {
+        [TestMethod]
+        [TestCategory("ProtocolTest")]
+        [TestCategory("RequestTest")]
+        [TestCategory("AwsQuery")]
+        public void QueryListsRequest()
+        {
+            var request = new QueryListsRequest{
+                ListArg =  new List<string>()
+                {
+                    "foo",
+                    "bar",
+                    "baz",
+                },
+                ComplexListArg =  new List<GreetingStruct>()
+                {
+                    new GreetingStruct{
+                        Hi = "hello",
+                    },
+                    new GreetingStruct{
+                        Hi = "hola",
+                    },
+                },
+            };
+            var config = new AmazonQueryProtocolConfig{
+              ServiceURL = "https://test.com/"
+            };
```

Comment:
Style throughout isn't consistent with other generated C# code in the repo, e.g.
* Opening braces are usually on next line, not same line
* Missing new lines between separate methods